extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientProcessor.java

```diff
@@ -114,7 +124,7 @@ BsonDiscriminatorBuildItem collectBsonDiscriminators(CombinedIndexBuildItem inde
      * Create a producer bean managing the lifecycle of the MongoClient.
      * <p>
      * The generated class will look like
-     * 
```

Comment:
Any reason this change was made or what it simply done by the formatter? Dittor for the similar changes below

Reply:
I think this might just have been to formatter, I will check.